Cargo.toml

```diff
@@ -10,3 +10,6 @@ repository = "https://github.com/timothee-haudebourg/pct-str"
 documentation = "https://docs.rs/pct-str"
 license = "MIT/Apache-2.0"
 readme = "README.md"
+
+[dependencies]
+utf8-decode = "1.0.1"
```

Comment:
Maybe we can relax the version constraint here, for instance:
```
utf8-decode = "^1.0"
```

Reply:
In Cargo.toml, "1.0" is equivalent to "^1.0". I could just make it "1" if you want anything <2.0.0

Reply:
"1" is fine.